components/Ink/src/private/MDCInkLayer.m

```diff
@@ -379,16 +432,40 @@ - (void)exit:(BOOL)animated {
   [self addAnimation:_backgroundOpacityAnim forKey:kInkLayerBackgroundOpacityAnim];
 }
 
+- (void)clearAnimations {
```

Comment:
Does it make sense to overwrite removeAllAnimations calling super?  Do they have subtly different behavior?